### PR TITLE
Feat add license

### LINKED ISSUE #15: add apache license
todo:
1 add apache license on every files;
2 add NOTICE as https://github.com/apache/dubbo-go/blob/3.0/NOTICE;

### PR DESCRIPTION
**Pull request by PangXing**
fix #15 
add license and readme.

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# seata-python: 简单的可扩展自主事务架构(Python版本)
+
+[![Build Status](https://github.com/seata/seata/workflows/build/badge.svg?branch=develop)](https://github.com/seata/seata/actions)
+[![license](https://img.shields.io/github/license/seata/seata.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+
+[English 🇺🇸](./README.md)
+
+## 什么是 seata-python？
+
+Seata是一个非常成熟的分布式事务框架，在Java领域是事实上的分布式事务技术标准平台。Seata-python 是 seata 多语言生态中的Go语言实现版本，实现了 Java 和 Python 之间的互通，让 Go 开发者也能使用 seata-python 来实现分布式事务。请访问[Seata 官网](https://seata.io/zh-cn/)查看快速开始和文档。
```
Comment (jsbxyyx) [inline README_CN.md:11]: Go 改成 Python

File: flask-sample/db.py [hunk 2]
```diff
@@ -1,6 +1,19 @@
-#!/usr/bin/env python3
-# -*- coding:utf-8 -*-
-# @author jsbxyyx
+"""
```
Comment (jsbxyyx) [inline flask-sample/db.py:1]: 缺少 # -*- coding: utf-8 -*-

File: README_CN.md (hunk 1 above)
Comment (PangXing) [reply README_CN.md]: Done.

File: flask-sample/db.py (hunk 2 above)
Comment (PangXing) [reply flask-sample/db.py:3]: Done.

**Review by AlexStocks — APPROVED** [review]
LGTM

**Review by jsbxyyx — APPROVED** [review]
LGTM

**Pull request merged**